### ISSUE TITLE
Support separate insights_config for postgresql read replicas

### ISSUE THREAD

**Issue by jawnsy** [terraform-google-modules/terraform-google-sql-db#483]
### TL;DR

The MySQL module allows read replicas to have a separate insights_config, which overrides the default: https://github.com/terraform-google-modules/terraform-google-sql-db/blob/2fcad8c54193a4c564b26d4bd8fba2b7ec788b5c/modules/mysql/variables.tf#L255-L259

However, the PostgreSQL module does not: https://github.com/terraform-google-modules/terraform-google-sql-db/blob/2fcad8c54193a4c564b26d4bd8fba2b7ec788b5c/modules/postgresql/variables.tf#L235-L264

It's unclear whether this is a limitation of the database type or just the module. The PostgreSQL module seems to inherit the configuration from the primary: https://github.com/terraform-google-modules/terraform-google-sql-db/blob/2fcad8c54193a4c564b26d4bd8fba2b7ec788b5c/modules/postgresql/read_replica.tf#L64

Whereas the MySQL module uses the replica-specific setting (this should probably inherit settings from the primary, and only override in replicas if set, via the Terraform merge function or similar): https://github.com/terraform-google-modules/terraform-google-sql-db/blob/2fcad8c54193a4c564b26d4bd8fba2b7ec788b5c/modules/mysql/read_replica.tf#L46
